server/planning/events/events_template.py

```diff
@@ -39,6 +39,7 @@ class EventsTemplateResource(Resource):
     }
     _event_fields = {
         "slugline": {"type": "string", "required": False, "readonly": True},
+        "language": {"type": "string", "required": False, "readonly": True},
```

Comment:
It might be easier to maintain EventTemplates if we don't restrict the resource schema to specific fields, but rather allow any fields to be stored under the `data` attribute.